deckhouse-controller/pkg/controller/module-controllers/utils/utils.go

```diff
@@ -38,3 +43,8 @@ func GenerateRegistryOptions(ms *v1alpha1.ModuleSource) []cr.Option {
 
 	return opts
 }
+
+// CalculateChecksum calculates md5 hash of .spec.registry relevant settings of a module source
+func CalculateModuleSourceChecksum(registryCfg *v1alpha1.ModuleSourceSpecRegistry) string {
```

Comment:
This is not a ModuleSource checksum, but a spec registry checksum. The name is misleading, the developers may think that they can use the method to find the checksum for the ModuleSource itself.

Reply:
removed